wrappers/javascript/react-native-app/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "react-native-app",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "expo start --dev-client",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "build": "exit 0",
```

Comment:
I usually do "echo 'no build needed for react-native-app'" or something similar

Reply:
Yeah I did `node -e console.log...` before (Windows does not have echo I believe), but that did not work. if we can do echo for windows I am okay with that.

Reply:
ah yeah not sure tbh. Let's stick with this